index.html

```diff
@@ -9,55 +9,44 @@
 </head>
 <body>
     <section class="header">
-        <nav>
-            <div class="logo"><p>My Logo</p></div>
-            <div class="menu"><img src="img/Union.png"></div>
+    <header>
+        <nav class="MenuBar">
+            <div class="logoPlace"> 
+                <a href="index.html" class="logo">
+                        <p>My Logo</p>
+                </a>
```

Comment:
- [Optional] Great Job for wrapping the hamburger with `nav`, I think it would be better if you move the logo outside the `nav` as it is not part of the navigation 👍